README.md

```diff
@@ -1,103 +1,47 @@
-GOV.UK Publishing Components
+# GOV.UK Publishing Components
+
 [![status](https://badgen.net/github/status/alphagov/govuk_publishing_components/master)](https://ci.integration.publishing.service.gov.uk/job/govuk_publishing_components/job/master/)
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
 [![Stylelint Style Guide](https://img.shields.io/badge/code_style-stylelint-brightgreen.svg)](https://github.com/alphagov/stylelint-config-gds/)
-=====================
 
-GOV.UK Publishing Components is a Ruby gem to document and distribute components for GOV.UK applications.
+A Ruby gem to document and distribute components for GOV.UK applications. It provides:
 
-It provides:
 - a shared library of components
 - helpers to generate component payloads
 - an application to preview components and provide guidance
+- [GOV.UK Design System](https://design-system.service.gov.uk/) styles and components (via [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend))
 
-Components should be added to this gem if they are required in more than one application, otherwise they should be added to that application.
-
-## How to
-
-- [Install and use this gem](/docs/install-and-use.md)
-- [Use a component in your application](/docs/use-components.md)
-- [Generate a new component](/docs/generate-a-new-component.md)
-- [Develop a component](/docs/develop-component.md)
-- [Run the component guide](/docs/run-component-guide.md)
-- [Move a component from an application to the gem](/docs/moving-components-upstream-into-this-gem.md)
-- [Publish/release a new version of the gem](/docs/publishing-to-rubygems.md)
-- [Keep this gem in sync with the Design System](/docs/upgrade-govuk-frontend.md)
-
-## Architecture / structure
-
-![](https://docs.google.com/drawings/d/e/2PACX-1vRj6JM7cQvngDl3Gr_U9G4xga2gsU7Z-d2qHHQcsBdjsW4WaC9_eQdryBJIS69cLkrY7S0fK9BcrPSF/pub?w=960&amp;h=720)
-
-[Source](https://docs.google.com/drawings/d/1N8-kbyCN_xOvvshN6d2HnQz5i5Bqed2WIatI3Nj9gNQ/edit)
-
-There are 2 types of helper classes in this app:
-
-- [AppHelpers](lib/govuk_publishing_components/app_helpers). Are exposed to the applications using this gem. They should be documented using RDoc.
-- [Component Presenters](lib/govuk_publishing_components/presenters). Anything in these classes is only for use within the components. They should be marked `@private`.
+Components should be added to this gem if they are required in more than one application, otherwise they should be added to that application. All components should meet the same standards of browser and assistive technology support described in [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend#browser-and-assistive-technology-support).
 
-## Relationship with GOV.UK Design System
+- 👉 [Install and use this gem](docs/install-and-use.md)
+- 👉 [Use a component in your application](docs/use-components.md)
+- 👉 [Develop a component](docs/develop-component.md)
```

Comment:
What is the reasoning for splitting the documentation links into two?

Reply:
I've removed the original commit and replaced it with a new one to do this: 727ec0cdac4b8bd1b717564b340b7688fb3f3a59